### PR TITLE
Add warcraft 2000

### PR DESCRIPTION
**Pull request by cxong**

### CODE REVIEW

File: originals/w.yaml [hunk 1]
```diff
@@ -1,3 +1,18 @@
+- name: ' Warcraft 2000'
+  names:
+  - ' Warcraft 2000: Nuclear Epidemic'
+  external:
+    website: 'https://www.myabandonware.com/game/warcraft-2000-nuclear-epidemic-bip'
```
Comment (cxong) [inline originals/w.yaml:5]: don't know if this is the best website - there isn't an official page for this game

File: originals/w.yaml [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+  platform:
+  - Windows
+  meta:
+    genre:
+    - RTS
```
Comment (pehsa) [inline originals/w.yaml:10]:
```suggestion
    - Real-Time Strategy
```

File: originals/w.yaml [hunk 3]
```diff
@@ -1,3 +1,18 @@
+- name: ' Warcraft 2000'
+  names:
+  - ' Warcraft 2000: Nuclear Epidemic'
```
Comment (pehsa) [inline originals/w.yaml:3]:
```suggestion
  - 'Warcraft 2000: Nuclear Epidemic'
```

File: originals/w.yaml [hunk 4]
```diff
@@ -1,3 +1,18 @@
+- name: ' Warcraft 2000'
```
Comment (pehsa) [inline originals/w.yaml:1]:
```suggestion
- name: 'Warcraft 2000'
```

File: games/w.yaml [hunk 5]
```diff
@@ -46,6 +46,29 @@
   video:
     youtube: vL0S_zVVhts
 
+- name: ' Warcraft 2000: Nuclear Epidemic'
+  originals:
+  - ' Warcraft 2000'
```
Comment (pehsa) [inline games/w.yaml:51]:
```suggestion
  - 'Warcraft 2000'
```

File: games/w.yaml [hunk 6]
```diff
@@ -46,6 +46,29 @@
   video:
     youtube: vL0S_zVVhts
 
+- name: ' Warcraft 2000: Nuclear Epidemic'
```
Comment (pehsa) [inline games/w.yaml:49]:
```suggestion
- name: 'Warcraft 2000: Nuclear Epidemic'
```

**Pull request merged**